web_accounts_list.json

```diff
@@ -2807,6 +2807,17 @@
          "known_accounts" : ["alice","bob"],
          "category" : "shopping",
          "valid" : true
+      },
+      {
+         "name" : "Fabswingers",
+         "check_uri" : "https://www.fabswingers.com/profile/{account}",
+         "account_existence_code" : "200",
+         "account_existence_string" : "View Profile",
+         "account_missing_string" : "The user you tried to view doesn't seem to be on the site any more",
+         "account_missing_code" : "200",
+         "known_accounts" : ["fabswingers.com"],

```

Comment:
Would you mind changing this value from `fabswingers.com` (which looks like a fake account because it is a domain) to some other, real accounts like `foreveryours87` and `viciqueen`? Or at least add them to the list of `known_accounts`?

Reply:
This is the admin/owners account so I choose that one since that one wouldn't disappear as easily as a random one I could find.

Reply:
Makes sense. Would you mind adding 1 or 2 other "user" accounts?

Reply:
Sorry - wasn't logged in to GH on the host I made the changes from so couldn't reply but they should be good now.